docs/docs/userguide/CONFIGURATION.md

```diff
@@ -194,6 +194,16 @@ db_allocated_storage = 100
 db_iops = 1000
 ```
 
+### Number of Bamboo agents
+`number_of_bamboo_agents` sets the number of remote agents to be launched. 
+
+```terraform
+number_of_bamboo_agents = 50
+```
+
+* Note: This number should not be greater than the value of Edition in the issued license. 
```

Comment:
I think `Edition` is an internal concept, this should be something along the lines:

```suggestion
* Note: This number should not be greater than the number of allowed remote agents in your license.
```
<img width="662" alt="Screen Shot 2021-12-21 at 4 15 29 pm" src="https://user-images.githubusercontent.com/416238/146875293-28169c41-309b-47d5-b088-13950c2f1ccc.png">